### DISCUSSION TITLE
logback.xml support

### DISCUSSION THREAD

**Discussion by max-peroch** [outr/scribe#608] [Q&A] [answered]
Hello, I was wondering if Scribe supports reading logback.xml files for formatting? I haven't been able to figure it out. Thanks

**Comment by darkfrog26** [comment, marked as answer]
No, we have basic support for log4j configuration loading, but most companies prefer to re-do their configuration programmatically or with their configuration instead of continuing to use legacy logging config.

I would be happy to help you out if you are interested in contributing to this feature, though.
